easybuild/easyconfigs/t/tn93/tn93-1.0.7-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,48 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Iñaki Mtz de Ilarduya Muñoz
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = "CMakeMake"
+
+name = 'tn93'
```

Comment:
@i-mtz from the github page it looks like the official name is in capital letters: `TN93`

Reply:
I have renamed the name inside and also the easyconfig, however I'm not sure about how to remove the old tn93 easyconfig from the pr (besides from the GitHub interface). Is there a way to do it from the `eb --update-pr`?

Reply:
you can remove an easyconfig by prefixing with a colon:
```
eb --update-pr 14691 --pr-commit-msg <message> TN93-1.0.7-GCCcore-10.3.0.eb :tn93-1.0.7-GCCcore-10.3.0.eb
```

Reply:
Ah cool thanks for the tip, just updated the PR without the old lowercase easyconfig.